### PR TITLE
Ownerstats

### PR DESCRIPTION
**Pull request by WallyThePenguin**
OwnerStats.

### CODE REVIEW

File: commands/Admins/Settings/ownerstats.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const { Command, version: klasaVersion, Duration } = require('klasa');
+const { version: discordVersion, MessageEmbed } = require('discord.js')
+const { prefix } = require('../../../../config/config.json');
```
Comment (Skillz4Killz) [inline commands/Admins/Settings/ownerstats.js]: What is this? Remove this line. We just need to use this.client.options.prefix;
```suggestion

```

File: commands/Admins/Settings/ownerstats.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const { Command, version: klasaVersion, Duration } = require('klasa');
+const { version: discordVersion, MessageEmbed } = require('discord.js')
+const { prefix } = require('../../../../config/config.json');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			guarded: true,
```
Comment (Skillz4Killz) [inline commands/Admins/Settings/ownerstats.js]: Why is this guarded?
```suggestion
```

File: commands/Admins/Settings/ownerstats.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+const { Command, version: klasaVersion, Duration } = require('klasa');
+const { version: discordVersion, MessageEmbed } = require('discord.js')
+const { prefix } = require('../../../../config/config.json');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			guarded: true,
+			description:'Here, the bot owner can look over his stats, and other things he needs.',
```
Comment (Skillz4Killz) [inline commands/Admins/Settings/ownerstats.js]:
```suggestion
			description: 'See the bots private stats that users don't need.',
```

File: commands/Admins/Settings/ownerstats.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+		.addField(`Memory Usage:`,  (process.memoryUsage().heapUsed / 1024 / 1024).toFixed(2),true)
+		.addField(`Uptime:`, Duration.toNow(Date.now() - (process.uptime() * 1000)),true)
+		.addField(`Server count:`, `${this.client.guilds.size}`,true)
+		.addField(`Player/User count:`,` ${this.client.users.size}`,true)
+		.addField(`Owner:`, (this.client.owner),true)
+		.addField(`Default Prefix:`,(prefix),true)
+		.addField(`Klasa Version:`,klasaVersion,true)
+		.addField(`Discord.js Version:`, discordVersion,true)
+		.addField(`Node.js Version`, process.version,true)
+		message.send(embed1)
```
Comment (Skillz4Killz) [inline commands/Admins/Settings/ownerstats.js]:
```suggestion
		message.send(new MessageEmbed()
		.setAuthor(this.client.user.username, this.client.user.displayAvatarURL())
		.setColor(7415665)
		.addField(`Memory Usage:`,  (process.memoryUsage().heapUsed / 1024 / 1024).toFixed(2),true)
		.addField(`Uptime:`, Duration.toNow(Date.now() - (process.uptime() * 1000)),true)
		.addField(`Server count:`, `${this.client.guilds.size}`,true)
		.addField(`Player/User count:`,` ${this.client.users.size}`,true)
		.addField(`Owner:`, (this.client.owner),true)
		.addField(`Default Prefix:`,(prefix),true)
		.addField(`Klasa Version:`,klasaVersion,true)
		.addField(`Discord.js Version:`, discordVersion,true)
		.addField(`Node.js Version`, process.version,true))
```

File: commands/Admins/Settings/ownerstats.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+			guarded: true,
+			description:'Here, the bot owner can look over his stats, and other things he needs.',
+			permissionLevel: 10,
+			aliases: ['os']
+		});
+	}
+
+	async run(message) {
+		
+	const embed1 = new MessageEmbed()
```
Comment (Skillz4Killz) [inline commands/Admins/Settings/ownerstats.js]: If you dont reuse the embed variable multiple times not much sense in making it a variable at all.

Comment (Skillz4Killz) [reply commands/Admins/Settings/ownerstats.js]: delete all this

**Comment by WallyThePenguin** [comment]
Go ahead now

**Pull request merged**